packages/perseus/src/widgets/radio/__tests__/radio.test.ts

```diff
@@ -874,3 +879,108 @@ describe("multi-choice question", () => {
         },
     );
 });
+
+describe("scoring", () => {
+    let userEvent: UserEvent;
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    /**
+     * (LEMS-2435) We want to be sure that we're able to score shuffled
+     * Radio widgets outside of the component which means `getUserInput`
+     * should return the same order that the rubric provides
+     */
+    it("can be scored correctly when shuffled", async () => {
+        // Arrange
+        const {renderer} = renderQuestion(shuffledQuestion);
+
+        // Act
+        await userEvent.click(
+            screen.getByRole("radio", {name: /Correct Choice$/}),
+        );
+
+        const userInput = renderer.getUserInput()[0] as PerseusRadioUserInput;
+        const rubric = shuffledQuestion.widgets["radio 1"].options;
+        const score = radioValidator(userInput, rubric, mockStrings);
+
+        // Assert
+        expect(score).toHaveBeenAnsweredCorrectly();
+        expect(renderer).toHaveBeenAnsweredCorrectly();
+    });
+
+    /**
+     * (LEMS-2435) We want to be sure that we're able to score shuffled
+     * Radio widgets outside of the component which means `getUserInput`
+     * should return the same order that the rubric provides
+     */
+    it("can be scored incorrectly when shuffled", async () => {
+        // Arrange
+        const {renderer} = renderQuestion(shuffledQuestion);
+
+        // Act
+        await userEvent.click(
+            screen.getByRole("radio", {name: /Incorrect Choice 1$/}),
+        );
+
+        const userInput = renderer.getUserInput()[0] as PerseusRadioUserInput;
+        const rubric = shuffledQuestion.widgets["radio 1"].options;
+        const score = radioValidator(userInput, rubric, mockStrings);
+
+        // Assert
+        expect(score).toHaveBeenAnsweredIncorrectly();
+        expect(renderer).toHaveBeenAnsweredIncorrectly();
+    });
+
+    /**
+     * (LEMS-2435) We want to be sure that we're able to score shuffled
+     * Radio widgets outside of the component which means `getUserInput`
+     * should return the same order that the rubric provides
+     */
+    it("can be scored correctly when shuffled with none of the above", async () => {
+        // Arrange
+        const {renderer} = renderQuestion(shuffledNoneQuestion);
+
+        // Act
+        await userEvent.click(
+            screen.getByRole("radio", {name: /None of the above$/}),
+        );
+
+        const userInput = renderer.getUserInput()[0] as PerseusRadioUserInput;
+        const rubric = shuffledNoneQuestion.widgets["radio 1"].options;
+        const score = radioValidator(userInput, rubric, mockStrings);
+
+        // Assert
+        expect(score).toHaveBeenAnsweredCorrectly();
+        expect(renderer).toHaveBeenAnsweredCorrectly();
```

Comment:
Are you asserting the `renderer` here just to ensure they both say the same thing?

Reply:
Yeah, I just figured "why not" - I already had the renderer since that was the simplest way I could think to test this.